rcon/scoreboard.py

```diff
@@ -442,15 +442,16 @@ def send_message(
         )
     else:
         logger.info(f"enqueuing {wh.message_id=} {key=}")
-        enqueue_message(
+        enqueue_scoreboard_message(
```

Comment:
would be nice to have a more generic function. Maybe something like `enqueue_message_last` or something to communicate that this method will discard older queued messages. And instead of message_key I would use the wording `message_group_key` or something.

message_key sounds like it is a unique id for that specific message, not for the group of messages where older messages are discarded.